### PR TITLE
Bubble Sort

### PR DESCRIPTION
**Pull request by rida-elbahtouri**
### Bubble Sort
**In this project we:**

- Build a method #bubble_sort that takes an array and returns a sorted array.

- created a similar method called #bubble_sort_by which sorts an array by accepting a block

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi Team 👋

You've both done pretty well on this. Kudos @rida-elbahtouri  and @kiptubei 🚀

However, there is an issue with your bubble_sort_by method. It fails to return a value when an array is passed to it, and a block is given.
Kindly cross-check your code to ensure you are returning a value.

Please fix this issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAVE A GREAT CODING DAY👨‍💻

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+def bubble_sort(array)
+  array_size = array.length
+  print array
+  puts ''
```
Comment (maelfosso) [inline bubble_sort.rb]: Remove that please

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# testing bubble_sort
+
+print bubble_sort([4, 6, 3, 6, 7, 8, 0])
+
+def bubble_sort_by(arr_param)
+  array_size = arr_param.length
+  puts ''
+  print arr_param
+  puts ''
```
Comment (maelfosso) [inline bubble_sort.rb:19]: Remove those lines please

File: bubble_sort.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    puts ''
+    return arr_param
+  end
+end
+
+# testing bubble_sort_by
+
+bubble_sort_by(%w[hi hello hey]) do |left, right|
+  left.length - right.length
+end
```
Comment (maelfosso) [inline bubble_sort.rb]: Not needed!

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: 

Thank you for fixing the bugs. You can merge it now
**PS**: Please, kindly read the comments below and apply the recommendations before merging.

Good continuation :handshake: 

Mael FOSSO

**Pull request merged**